.ci/containers/downstream-builder/Gemfile

```diff
@@ -0,0 +1,15 @@
+source 'https://rubygems.org'
```

Comment:
Is this supposed to be a carbon-copy of the Magic Modules gemfile? Can we symlink this to avoid it getting out of date?

Reply:
Yeah, I hated that.  Symlinks don't work in git repos, sadly, and docker context can't go up out of the context directory.  I now fetch those directly from github.